web/src/components/TaskFooter.tsx

```diff
@@ -0,0 +1,14 @@
+import { ComponentPropsWithoutRef, ReactElement } from "react";
+
+export const TaskFooter = (props: ComponentPropsWithoutRef<"div">): ReactElement => {
+  const { children, ...attributes } = props;
+  return (
+    <div
+      style={{ marginTop: "2rem" }}
```

Comment:
[dust] You can also add the class `margin-top-4` to achieve the same thing and avoid the inline style.

Reply:
Good catch, thanks!